src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -13,6 +13,10 @@
     <NoWarn>$(NoWarn);CS1591;CS8002</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
```

Comment:
We have far too many public APIs in the dashboard project. Why disable it here?

Reply:
Aspire.Dashboard isn't an API surface though, its a tool that we ship. Do we expect people to be taking a dependency on Aspire.Dashboard in their own projects?

Reply:
Then there should be little reason for the API to be public? ;)

Reply:
> Do we expect people to be taking a dependency on Aspire.Dashboard in their own projects?

No we don't. It doesn't ship as a package on nuget.org.

> Then there should be little reason for the API to be public?

Some things use reflection that require things to be `public`, it won't reflect on `internal` types. Having public types and methods in an .exe is not an issue.

Reply:
You'd also be fighting the default Razor code gen all day long.